docs/AboutUs.md

```diff
@@ -18,4 +18,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
+<<<<<<< HEAD
```

Comment:
Think you have forgotten to remove this line

Reply:
Amended and update my part for user guide as well